WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsFragment.java

```diff
@@ -853,7 +854,7 @@ private void updateFeaturedImageView() {
             mediaUri = PhotonUtils.getPhotonImageUrl(mediaUri, size, 0);
         }
 
-        WordPressMediaUtils.loadNetworkImage(mediaUri, mFeaturedImageView, mImageLoader);
```

Comment:
Now that `mImageLoader` is no longer used we can drop `@Inject FluxCImageLoader mImageLoader;`